src/main/java/TodoCommand.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Encapsulates a "todo" command from the user.
+ */
+public class TodoCommand implements Command {
```

Comment:
I'm not sure if TodoCommand should be written as such or ToDoCommand; I did the latter because of the camel case requirements. Perhaps you might want to consider changing it! Just a suggestion :) 

Otherwise, the code is well-written.